misc/perf-dash/create_views/create_views

```diff
@@ -11,5 +11,5 @@
 set -euo pipefail
 
 for view_file in *.sql; do
-    psql -h perf-dash-metrics -p 8675 -U materialize -f "${view_file}" materialize
+    psql -v ON_ERROR_STOP=1 -h perf-dash-metrics -p 8675 -U materialize -f "${view_file}" materialize
```

Comment:
Will this also make the scripts exit with `1` or does it just stop `psql` on the first error it encouters?

Reply:
Both the script and mzcompose will also exit with error!